modules/cnv-monitoring-upgrade-status.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_install/upgrading-container-native-virtualization.adoc
+
+[id="cnv-monitoring-upgrade-status_{context}"]
+= Monitoring upgrade status
+
+The best way to monitor {CNVProductName} upgrade status is to watch the
+ClusterServiceVersion (CSV) `PHASE`.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* The {product-title} CLI (`oc`) installed.
+
+.Procedure
+
+. Run the following command:
++
+----
+$ oc get csv
+----
+
+. Review the output, checking the `PHASE` field. For example:
++
+----
+VERSION  REPLACES                                  PHASE
+2.0.99                                             Installing
+2.1.0    kubevirt-hyperconverged-operator.v2.0.99  Pending
+----
++
+[WARNING]
+====
+`PHASE` represents the system's best guess based on available information.
+====
+
+. (Optional) Monitor the status of all {CNVProductName} components by running
```

Comment:
Monitor the aggregated status of all {CNVProductName} component `Conditions` by running